exercises/05.complex-structures/01.solution.nested/app/routes/users+/$username_+/notes.$noteId_.edit.tsx

```diff
@@ -200,7 +200,7 @@ function ImageChooser({
 	const [altText, setAltText] = useState(fields.altText.defaultValue ?? '')
 
 	return (
-		<fieldset ref={ref}>
+		<fieldset ref={ref} {...conform.fieldset(config)}>
```

Comment:
Was it intentional to omit the aria attributes?

Reply:
Yeah, just because I didn't want it to distract from things. It's great that those are added automatically now so we don't need to worry about it. This change is perfect 👍